packages/e2e-test-utils-playwright/src/request-utils/comments.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+export interface Comment {
+	id: number;
+	author: number;
+	content: string;
+}
+
+/**
+ * Create new comment using the REST API.
+ *
+ * @param {} this    RequestUtils.
+ * @param {} comment Comment.
+ */
+export async function createComment( this: RequestUtils, comment: Comment ) {
+	this.rest( {
+		method: 'POST',
+		path: '/wp/v2/comments',
+		data: comment,
```

Comment:
I don't think we have the `id` field of the comment data yet before creating it?

Reply:
Also, why couldn't we call `/wp/v2/users/me` here to get the current user id to pass it into the data? So that we don't have to create another util `getCurrentUser` for this.